### DISCUSSION TITLE
Paid organizations can now launch projects on bigger compute immediately

### DISCUSSION THREAD

**Discussion by kevcodez** [supabase/supabase#21386] [Changelog]
Paid plan users can now immediately launch projects on larger compute sizes. Previously, paid organizations had to launch projects on the default "Micro" instance and then separately upgrade their instance. You can always up- and downgrade your instance in hindsight.

![image](https://github.com/supabase/supabase/assets/14073399/d61d7a58-8b8e-4d9c-93e8-cd005d695cfe)

![image](https://github.com/supabase/supabase/assets/14073399/07a26ec0-2b89-45cf-8179-3eb00e45d305)

**Comment by madsbolaris** [comment]
Fixed in PR #1048 (merged). Added an explicit, sanctioned fallback path for when Primer-dev MCP tools are unavailable:

- **AGENTS.md** Step 1: full three-source fallback (components via `.d.ts`, tokens via `node_modules/@primer/primitives/dist/`, octicons via `data.json`), PR-description audit trail for both MCP and fallback paths
- **specs/quality-gates.spec.md** Gate 1: expanded with the same fallback format and example
- **.github/copilot-instructions.md**: CCR rule now flags PRs missing Primer-dev documentation
- **.github/instructions/css|react|typescript.instructions.md**: unified fallback references throughout

Thanks for filing this — the fallback path is now fully documented and consistent across all instruction files.
